_posts/2024-01-05-devcontainer-platform-agnostic-team.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog_post
+title: "How to support a platform-agnostic engineering team with VS Code Dev Containers"
+subtitle: "Learn from the Compiler Engineering team, which has been using VS Code Dev Containers daily across Windows, Linux and Mac on all of their projects."
+description: "Learn from the Compiler Engineering team, which has been using VS Code Dev Containers daily across Windows, Linux and Mac on all of their projects."
+author: Machiko Yasuda
+excerpt: "Learn from the Compiler Engineering team, which has been using VS Code Dev Containers daily across Windows, Linux and Mac on all of their projects."
+date:   2024-01-05 19:03:13 +0200
```

Comment:
the date? I am gonna update the date... to whenever the editors want to post it. I just had to put some kind of date in the past so that the preview link shows the blog post. (Makes it easier for others to review it)

Reply:
@machikoyasuda FYI, we will get you a date asap. I'm meeting with Milo today at 1:30pt to finalize our content calendar.

Reply:
It would be great to get this blog post out in sync with or around the same time as the new Software Engineer job posting goes out!

Reply:
ooh good thinking. noted!

Reply:
Which will be... sometime early next week according to our current plan!

Reply:
cc @laneymangan I just updated the date above!

Reply:
done